package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A full-stack implementation of the FerryTempo FTServer. Provides API endpoints and live debugging views.",
```

Comment:
Please make sure to bump the version number - this would be `1.2.0` according to semver: https://semver.org/

Reply:
Is this being incremented because of the addition of Jest? Or do we increment whenever this file is changed for any reason? Just want to make sure I am following convention.

Reply:
Ideally we increment on every change so that the server version is passed along with our data. This helps us when there's an issue with the output by letting us know if a specific change caused the issue.